docs/env.md

```diff
@@ -106,6 +106,45 @@ export ANNOUNCEMENT="Hello! I'm {USERNAME} and I'm an SSH bot! I'm currently lis
 export ANNOUNCEMENT="Hello! I'm {USERNAME} and I'm an SSH bot! Being in {CURRENT_TEAM} will grant you SSH access to certain servers. Reach out to @dworken for more information."
 ```
 
+## Two Man Realms
+
+It is possible to configure the SSH CA bot to require approval prior to granting access. For example, one could require
+that in order to use kssh with servers in the `team.ssh.root_everywhere` realm two other people must approve the request. 
+Approvals are done by reacting with a :+1: emoji to the message inside of Keybase chat. 
+
+In order to configure this, there are 3 environment variables that can be used:
+
+```
+export TWO_MAN_TEAMS="team.ssh.root_everywhere, team.ssh.ci"
+export TWO_MAN_APPROVERS="username0, username1, username2" 
+export TWO_MAN_APPROVER_COUNT="2"
+```
+
+The above will require that anyone who wants to use kssh with `team.ssh.root_everywhere` or `team.ssh.ci` gets approval
+from two other people. Only three people are configured to approve requests: `username0, username1, username2`. 
+
+A few other example configurations are:
+
+``` 
+export TWO_MAN_TEAMS="team.ssh.root_everywhere"
+export TWO_MAN_APPROVERS="username" 
+```
+
+This would require that in order to use kssh with `team.ssh.root_everywhere` it must be approved by `username`. 
+
+As a user, this would be done by running:
+
+```
+kssh --request-realm team.ssh.root_everywhere root@production
```

Comment:
this is kind of a mouthful. any chance it can work more like:

```
> kssh root@production
ERROR: awaiting approval in team.ssh.root_everywhere
...
> kssh root@production
```
OR
```
> kssh root@production
ERROR: you need to request approval first like: `kssh --request root@production`
> kssh --request root@production
...
> kssh root@production
```

Reply:
Sadly, there isn't really a way to make that work :/ 

In your first example, there is no way for kssh to know which servers need which principals. This becomes especially hard when talking about ssh configs which can define custom names like "production" for a hostname. 

The second example runs into essentially the same issue. While kssh could attempt to go first and then alert the user if the connection fails, there is no way for `kssh --request root@production` to know which principals to request. 

A feature that could be built out is allowing the user to configure this sort of thing on the client side. Eg:

```
kssh root@production   # permission denied
kssh --configure-realm team.ssh.root_everywhere root@production  # configures kssh to always request that principal when accessing the given host
kssh root@production  # works 
```

The difficulty with the local config strategy is that it will break in weird ways when using proxy commands or any more advanced SSH features. Eg `kssh -J root@production david@other`. kssh wouldn't see the `root@production` connection and wouldn't know to provision a certificate with that principal. So IMO it doesn't seem worth it to build this feature given how brittle it would end up being, but certainly open to it if you think it would still be useful.

Reply:
it seems really sub-optimal to me to push the onus of knowing how everything is set-up down to the kssh user. 
> In your first example, there is no way for kssh to know which servers need which principals
but doesn't the bot know? or if not, can we make it know with additional environment variables?
it looks like the message requesting approval is coming from the kssh user. can it come from the bot instead? this will make it much easier for an operations team to iterate on how they want this thing to work for them without having to push an update to all kssh users, which i imagine would be much more difficult.